app/core/Engine.ts

```diff
@@ -806,16 +920,32 @@ class Engine {
   }
 }
 
-let instance: Engine;
+/**
+ * Assert that the given Engine instance has been initialized
+ *
+ * @param instance Either an Engine instance, or null
```

Comment:
Typo in JSDoc comment:
```suggestion
 * @param instance - Either an Engine instance, or null
```